src/nvim/os/fs.c

```diff
@@ -120,9 +120,13 @@ static bool is_executable(const char_u *name)
     return false;
   }
 
-  if (S_ISREG(mode) && (S_IXUSR & mode)) {
-    return true;
-  }
+#if WIN32
+  // Windows does not have exec bit, just check if
+  // the file exists
+  return (S_ISREG(mode));
```

Comment:
Oops we don't have `os_access`, we use `uv_fs_access` directly. So never mind I guess.